src/mantle/mantle_cmd_buf.c

```diff
@@ -415,3 +415,25 @@ GR_VOID grCmdClearColorImageRaw(
 
     free(vkRanges);
 }
+
+GR_VOID grCmdSetEvent(
+    GR_CMD_BUFFER cmdBuffer,
+    GR_EVENT event)
+{
+    GrCmdBuffer* grCmdBuffer = (GrCmdBuffer*)cmdBuffer;
```

Comment:
add `LOGT` to trace function calls. see other functions above

Reply:
done